test/helpers/helpers.go

```diff
@@ -203,19 +195,12 @@ func CheckCatalogSourceConfigAndChildResourcesCreated(client test.FrameworkClien
 	return nil
 }
 
-// CheckCatalogSourceConfigAndChildResourcesDeleted checks that a CatalogSourceConfig
-// and it's child resources were deleted.
-func CheckCatalogSourceConfigAndChildResourcesDeleted(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
-	// Check that the CatalogSourceConfig was deleted.
-	resultCatalogSourceConfig := &marketplace.CatalogSourceConfig{}
-	err := WaitForNotFound(client, resultCatalogSourceConfig, namespace, cscName)
-	if err != nil {
-		return err
-	}
-
+// CheckCatalogSourceConfigChildResourcesDeleted checks that a CatalogSourceConfig's
+// child resources were deleted.
+func CheckCatalogSourceConfigChildResourcesDeleted(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
```

Comment:
If anyone has any suggestions on how to further distinguish `CheckCatalogSourceConfigChildResourcesDeleted` from `CheckCatalogSourceConfigAndChildResourcesDeleted` please share.

Reply:
Can't we just make them two functions? One that just checks that the CatalogSoureConfig itself is deleted, and one that checks the child resources? Why does one have to be a superset of the other?

Reply:
There's value add having a single call that ensures that a `csc` and all it's child resources are created.